.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @asford
\ No newline at end of file
+* @PawelLipski @asford
```

Comment:
Whoops error? 🤔 
Oh, I need write access to this repo lol

Reply:
> Oh! @asford hello again, I didn't know you're the maintainer here 😯

Howdy again!
Yes, I've been maintaining this feedstock.
It's pretty much on autopilot via the autotick bot, other than the python version update you've identified.

Happy to add you as a maintainer here, but please tag me on PRs etc for a bit before merging,

There are some slightly magical bits that you need to know:
https://conda-forge.org/docs/maintainer/updating_pkgs.html#maintaining-pkgs

and the general flow should we reasonably well documented as you open PRs.